pkg/config/systemd.go

```diff
@@ -29,20 +37,17 @@ func defaultCgroupManager() string {
 }
 
 func defaultEventsLogger() string {
-	if useSystemd() {
+	if useJournald() {
 		return "journald"
 	}
 	return "file"
 }
 
 func defaultLogDriver() string {
-	// If we decide to change the default for logdriver, it should be done here.
-	if useSystemd() {
-		return DefaultLogDriver
+	if useJournald() {
+		return "journald"
 	}
-
-	return DefaultLogDriver
-
+	return "k8s-file"
```

Comment:
Should you still set DefaultLogDriver to k8s-file, return it here, and then just system it to systems in the appropriate places?

Reply:
No because we want DefaultLogDriver to be indicated as journald now.